server/server.js

```diff
@@ -8,8 +8,9 @@ const BASE_API_URL = 'https://api.spaceflightnewsapi.net/v3';
 const ARTICLES_ROUTE = '/articles';
 const REPORTS_ROUTE = '/reports';
 const BLOGS_ROUTE = '/blogs';
+const DASHBOARD_ROUTE = '/dashboard'
```

Comment:
Is it possible to import these variables from src/app/config/consts.ts?

Reply:
I thought about it but I want to keep server and UI parts as separated as possible.
We can discuss how to manage it in the future.